views/entity.html

```diff
@@ -10,254 +10,233 @@
 
 {% block content %}
 <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-
-        <h1 class="govuk-heading-xl">Tell us about the overseas entity</h1>
-
-        {{ govukDetails({
-            summaryText: "What is an overseas entity?",
-            text: "An overseas entity is an entity which is a legal person in its home country."
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-xl">Tell us about the overseas entity</h1>
+
+    {{ govukDetails({
+      summaryText: "What is an overseas entity?",
+      text: "This is a legal entity, such as a company or organisation, that is governed by the law of a country or territory outside the UK."
+    }) }}
+
+    <form method="post">
+
+      {{ govukInput({
+        label: {
+          text: "What is the name of the overseas entity?",
+            classes: "govuk-label--m",
+            isPageHeading: false
+          },
+          id: "name",
+          name: "name",
+          value: name
+      }) }}
+
+      {{ govukInput({
+        label: {
+          text: "Which country was it formed in?",
+          classes: "govuk-label--m",
+          isPageHeading: false
+        },
+        id: "incorporation_country",
+        name: "incorporation_country",
+        value: incorporation_country
+      }) }}
+
+      {% include "includes/inputs/principal-address-input.html" %}
```

Comment:
As Mark pointed out there's two address radio options. The includes used above contains where you input the principal address, the radio buttons and the subsequent area to input the service address so you wont need the that bit just below here down to line 160 I believe.

Reply:
Have noticed that the text on the radio buttons in the includes element I made is incorrect and should be "Is the correspondence address the same as the principal or registered office address" so that needs to be changed, feel free to do it in this or I'll do it in my story

Reply:
I added the includes at the last minute and forgot to take the address options on the main page away.